webapp/src/ts/services/update-user.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from '@angular/common/http';
+import {Observable} from "rxjs";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UpdateUserService {
```

Comment:
@dianabarsan this is the old "UpdateUser" component but now that we use classes maybe it is best to rename to just "UserService", right? so we can add in the future related functions to the user here instead of create a new service.

Reply:
I think this was a service in the old Angular, not a component. 
````
angular.module('inboxServices').factory('UpdateUser',
```

Reply:
Sorry yes I wanted to say "service", not "component". Should I rename it to `UserService` ?